src/shared.py

```diff
@@ -128,26 +137,22 @@ def reloadMyAddressHashes():
         if isEnabled:
             hasEnabledKeys = True
             # status
-            _, addressVersionNumber, streamNumber, hash = \
+            _, addressVersionNumber, streamNumber, hashobj = \
```

Comment:
I remember this case. Now I think the right way was to write it like
```python
addressVersionNumber, streamNumber, hashobj = decodeAddress(addressInKeysFile)[1:]
```

Reply:
Oh, yes. This is the best way to solve this problem. I will change the code as you suggested.
Thank you.